inspire/base/templates/search/results.html

```diff
@@ -28,6 +28,53 @@
 {%- block search_result_settings_display %}
 {%- endblock search_result_settings_display %}
 
+{% block search_result_nb_recs scoped %}
+  {%- if pagination %}
+  <div class="row">
+    <div class="col-md-12">
+      <div class="panel panel-default" >
+        <div class="panel-body" >
+          <div class="form-group" style="display:block;">
+            <p class="help-block hidden-xs" style="font-size: 20px;">
+              {%- set r_from = (pagination.page-1)*pagination.per_page+1 -%}
+              {%- set r_to = pagination.page*pagination.per_page -%}
+              {%- set r_of = pagination.total_count -%}
+              {%- set r_to = r_to if r_to < r_of else r_of -%}
+              {# _('Showing records %(x_from)d to %(x_to)d out of %(x_total)d results.', x_from=r_from, x_to=r_to, x_total=r_of) #}
+              {{ _('<b style="color:#e57e25;">%(x_total)d</b> results found.', x_total=r_of) }}
+            </p>
+          </div>
+          <hr>
+          {% if  request.args.get('cc') not in ["HepNames", "Institutions", "Conferences", "Jobs", "Journals", "Experiments"] %}
```

Comment:
It is better to just check ```{% if  request.args.get('cc') == "HEP" %}```

As that is the only collection that should have the Export.

Otherwise if we add a new collection the dropdown will appear unexpectedly.